sdk/tracing/azure-core-tracing-opencensus/src/samples/java/com/azure/core/tracing/opencensus/PublishEvents.java

```diff
@@ -0,0 +1,90 @@
+import com.azure.messaging.eventhubs.EventHubAsyncClient;
+import com.azure.messaging.eventhubs.EventHubAsyncProducer;
+import com.azure.messaging.eventhubs.EventHubClientBuilder;
+import io.opencensus.common.Scope;
+import io.opencensus.exporter.trace.zipkin.ZipkinTraceExporter;
+import io.opencensus.trace.Tracer;
+import io.opencensus.trace.Tracing;
+import io.opencensus.trace.config.TraceConfig;
+import io.opencensus.trace.config.TraceParams;
+import io.opencensus.trace.samplers.Samplers;
+import reactor.core.publisher.Flux;
+
+import java.io.IOException;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+/**
+ * Sample demonstrates how to send a message to an Azure Event Hub with tracing support.
+ */
+public class PublishEvents {
+    /**
+     * Main method to invoke this demo on how to send a message to an Azure Event Hub
+     * with trace spans exported to zipkin.
+     *
+     * Please refer to the  <a href=https://zipkin.io/pages/quickstart>Quickstart Zipkin</a>
+     * for more documentation on using a zipkin exporter.
+     *
+     * @param args Unused arguments to the program.
+     */
+    public static void main(String[] args) {
+        ZipkinTraceExporter.createAndRegister("http://localhost:9411/api/v2/spans", "tracing-to-zipkin-service");
+
+        TraceConfig traceConfig = Tracing.getTraceConfig();
+        TraceParams activeTraceParams = traceConfig.getActiveTraceParams();
+        traceConfig.updateActiveTraceParams(activeTraceParams.toBuilder().setSampler(Samplers.alwaysSample()).build());
+
+        Tracer tracer = Tracing.getTracer();
+        String connectionString = "Endpoint=sb://eventdatamigrationns.servicebus.windows.net/;SharedAccessKeyName=samVPolicy;SharedAccessKey=t6fQstSM9lcH/51GJaS1XKXEnl3nWX1zcJJMviBadYI=;EntityPath=samv";
+
+
+        try (Scope scope = tracer.spanBuilder("opencensus-span").startScopedSpan()) {
+            // The connection string value can be obtained by:
+            // 1. Going to your Event Hubs namespace in Azure Portal.
+            // 2. Creating an Event Hub instance.
+            // 3. Creating a "Shared access policy" for your Event Hub instance.
+            // 4. Copying the connection string from the policy's properties.
+
+            // Instantiate a client that will be used to call the service.
+            EventHubAsyncClient client = new EventHubClientBuilder()
+                .connectionString(connectionString)
+                .buildAsyncClient();
+
+            final String CONTENTS = "SSLorem ipsum dolor sit amet, consectetur adipiscing elit. Donec vehicula posuere lobortis. Aliquam finibus volutpat dolor, faucibus pellentesque ipsum bibendum vitae. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Ut sit amet urna hendrerit, dapibus justo a, sodales justo. Mauris finibus augue id pulvinar congue. Nam maximus luctus ipsum, at commodo ligula euismod ac. Phasellus vitae lacus sit amet diam porta placerat. \n"
```

Comment:
I'd shorten the contents to something else like "Hello world". Having it this long takes away from the sample itself. Also, the variable name should be lower case?

Reply:
Changed to that.